### DISCUSSION TITLE
html elements or components in a table component

### DISCUSSION THREAD

**Discussion by officialankan** [skeletonlabs/skeleton#2514] [Q&A] [answered]
Hi guys!

Loving Skeleton UI so far. 

I'm trying out the table component (https://www.skeleton.dev/components/tables) and loving the simplicity. Has anyone figured out a neat way of inserting elements or components, like `<img />` or `<Avatar />`, inside a table? 

Thanks!

**Comment by Sarenor** [comment, marked as answer]
Either by building a ``<table>`` yourself using the ``.table`` class or by using the datatables we recommend: https://www.skeleton.dev/docs/ssd

Edited for future reference: The ``.table`` class: https://www.skeleton.dev/elements/tables

**Reply by officialankan** [reply]
Thanks! Coulnd't really figure out the table class but now found it thanks to the next comment. Works like a charm!

**Comment by endigo9740** [comment]
Here's the link to the Table elements Sereno mentioned. This will work for you:
https://www.skeleton.dev/elements/tables

The datatables guide is only needed if you need datatable features.
